data/categories.txt

```diff
@@ -4883,47 +4883,48 @@ th:โสตวิทยา|นักโสตสัมผัสวิทยา
 tr:Odyoloji|Odyolog
 uk:Аудіологія|Аудіолог
 vi:Thính học
-zh-Hans:听力学|听力学家
+zh-Hans:听力学家
 zh-Hant:聽力學|聽力學家
 
-healthcare-blood_donation
-en:Blood donation center|Blood donation
-ar:مركز التبرع بالدم|تبرع بالدم
-be:Цэнтр донарства крыві|Донарства крыві
-bg:Център за кръводаряване
-cs:Centrum dárcovství krve
-da:Center for bloddonation|Bloddonation
-de:Blutspendezentrum|Blutspende
-el:Κέντρο αιμοδοσίας|Αιμοδοσία
-es:Centro de donación de sangre|Donación de sangre
-et:Vereloovutuskeskus|Veredoonorlus
-eu:Odol Emate Zentroa|Odol ematea
-fa:مرکز اهدای خون|اهدای خون
-fi:Verenluovutuskeskus|Verenluovutus
-fr:Centre de don du sang|Don du sang
-he:מרכז תרומת דם|תרומת דם
-hu:Véradó központ|Véradás
-id:Pusat Donor Darah|Donor darah
-it:Centro di donazione del sangue|Donazione di sangue
-ja:献血センター|献血
-ko:헌혈센터|헌혈
-mr:रक्तदान केंद्र|रक्तदान
-nb:Bloddonasjonssenter|Bloddonasjon
-nl:Bloeddonatiecentrum|Bloeddonatie
-pl:Centrum krwiodawstwa|Oddawanie krwi
-pt:Centro de Doação de Sangue|Doação de sangue
-pt-BR:Hemocentro|Doação de sangue
-ro:Centrul de donare de sânge|Donarea de sânge
-ru:Центр донорства крови|Донорство крови
-sk:Centrum darcovstva krvi|Darovanie krvi
-sv:Centrum för blodgivning|Blodgivning
-sw:Kituo cha Uchangiaji Damu|Utoaji wa damu
-th:ศูนย์รับบริจาคโลหิต|การบริจาคเลือด
-tr:Kan Bağışı Merkezi|Kan Bağışı
-uk:Центр донорства крові|Донорство крові
-vi:Trung tâm hiến máu|Hiến máu
-zh-Hans:献血中心
-zh-Hant:捐血中心|獻血
+# This category tokens will interfere with other categories (due to the current search implementation).
```

Comment:
It can be just blood donation for the search. Later we'll implement automatic suggestions even without numbers.